demo-project/src/demo_project/pipelines/reporting/nodes.py

```diff
@@ -56,7 +58,10 @@ def make_price_histogram(model_input_data: pd.DataFrame) -> go.Figure:
         BaseFigure: Plotly object which is serialised as JSON for rendering
     """
     price_data_df = model_input_data[["price", "engine_type"]]
-
+    random_limit = random.randint(0, random.randint(5, 9)) * 1000
+    price_data_df["price"] = price_data_df["price"].apply(
+        lambda x: (x + random.randint(0, random_limit) * random.randint(0, 1))
```

Comment:
Similarly I'm a bit confused here about why we need to combine different numbers. What are we actually trying to achieve for the final `price` column?